runtime/compiler/control/CompilationThread.cpp

```diff
@@ -7405,6 +7406,7 @@ TR::CompilationInfoPerThreadBase::cannotPerformRemoteComp(
 #endif /* defined(J9VM_OPT_CRIU_SUPPORT) */
           !JITServer::ClientStream::isServerCompatible(OMRPORT_FROM_J9PORT(_jitConfig->javaVM->portLibrary)) ||
           (!JITServerHelpers::isServerAvailable() && !JITServerHelpers::shouldRetryConnection(OMRPORT_FROM_J9PORT(_jitConfig->javaVM->portLibrary))) ||
+	  (!JITServer::CommunicationStream::shouldReadRetry()) ||
```

Comment:
`shouldReadRetry()` does not use any exponential back-off mechanism. Was that the intent?